Documentation/guides/faq.html.md

```diff
@@ -106,7 +106,7 @@ can DM [orta][].
 
 [77]: https://github.com/danger/danger-js/issues/77
 [529]: https://github.com/danger/danger-js/issues/529
-[vision.md]: https://github.com/danger/swift/blob/master/VISION.md
+[vision.md]: https://github.com/danger/danger/blob/master/VISION.md
```

Comment:
```suggestion
[vision.md]: https://github.com/danger/danger-js/blob/master/VISION.md
```

We probably want this, though @f-meloni  - maybe you could write one for this repo now that it is stable?